quickmenu/arm9/source/language.inl

```diff
@@ -29,3 +29,5 @@ STRING(B_BACK, "\\B Back")
 STRING(START_FAILED_ERROR, "Start failed. Error %d")
 STRING(BOOTSTRAP_NIGHTLY_NOT_FOUND, "nds-bootstrap (Nightly)\nnot found.")
 STRING(BOOTSTRAP_RELEASE_NOT_FOUND, "nds-bootstrap (Release)\nnot found.")
+STRING(BOOTSTRAP_HB_NIGHTLY_NOT_FOUND, "nds-bootstrap for homebrew\n(Nightly) not found.")
```

Comment:
Can you add these to both of the english ini files too? Crowdin won't pick them up if they're only in here

Reply:
Fixed.